### PR TITLE
Breadcrumb

### PR DESCRIPTION
**Pull request by bromiesTM**

### CODE REVIEW

File: css/files.css [hunk 1]
```diff
@@ -1,3 +1,30 @@
+div.files-list__header:has(div.files-list__breadcrumbs) {
```
Comment (bromiesTM) [inline css/files.css:1]: todo: the breadcrumb in the target dialog is not yet customized

Comment (fracado) [reply css/files.css:1]: My suggestion to make it work for target dialog:

```
div.files-list__header:has(div.files-list__breadcrumbs), .file-picker {
	div.files-list__breadcrumbs, .file-picker__breadcrumbs {
		ul.breadcrumb__crumbs {
			a, button:not(.button-vue--icon-only) {

			}
		}
	}
}
```

**Review by fracado — COMMENTED** [review]
- breadcrumbs in target dialogs missing (see comment)
- needs to be checked again for fixes needed after #14 was merged

**Review by fracado — COMMENTED** [review]
Review OK, if you agree with suggestion.

**Pull request merged**